app/templates/base-flat.jade

```diff
@@ -48,6 +48,9 @@ mixin accountLinks
                 .nav-spacer
                 if !me.hideTopRightNav()
                   ul.nav.navbar-nav
+                    if me.isAnonymous() && require('app/core/experiments').getRequestAQuoteGroup(me) === 'request-a-quote-header'
+                      li
+                        a.text-p(data-event-action="Header Request Quote CTA", data-i18n="new_home.request_quote")
```

Comment:
Should something happen when this is clicked? From my basic understanding, we are getting an event here but there is no listener for `text-p` onClick? Or is this just meant to be displayed and not clicked?

Reply:
I think there is a generic click handler for `a` tags in RootView (I think, dont remember clearly) which tracks the event action, but worth checking this once to make sure.

Reply:
Yikes.  I forgot to add the href when I switched from an event driven implementation (to send the event with proper tag) to the global event tag with a non event driven implementation.  Thank you.

Reply:
Updated and will verify prior to ship